### PR TITLE
Created server folder, basic index.js file, and script to package.json

### PR DESCRIPTION
**Pull request by jabrake**
Also testing out requesting reviews for my repo @farhanali3193 @tchitrakorn

### CODE REVIEW

**Review by farhanali3193 — APPROVED** [review]

**Comment by farhanali3193** [comment]
@jabrake @tchitrakorn  Do we want to merge after everyone has approved? Or is one person's approval enough?

**Comment by jabrake** [comment]
Thanks! I think one person's approval should be enough.

**Pull request merged**
